chef/cookbooks/swift/templates/default/proxy-server.conf.erb

```diff
@@ -316,10 +320,20 @@ signing_dir = /var/cache/swift/keystone-signing
 use = egg:swift#keystoneauth
 # Operator roles is the role which user would be allowed to manage a
 # tenant and be able to create container or give ACL to others.
-operator_roles = Member,admin
+operator_roles = Member, admin
 # The reseller admin role has the ability to create and delete accounts
 # reseller_admin_role = ResellerAdmin
 reseller_prefix=<%= @reseller_prefix %>
+# For backwards compatibility, keystoneauth will match names in cross-tenant
+# access control lists (ACLs) when both the requesting user and the tenant
+# are in the default domain i.e the domain to which existing tenants are
+# migrated. The default_domain_id value configured here should be the same as
+# the value used during migration of tenants to keystone domains.
+# default_domain_id = default
+# For a new installation, or an installation in which keystone projects may
+# move between domains, you should disable backwards compatible name matching
+# in ACLs by setting allow_names_in_acls to false:
+# allow_names_in_acls = true
```

Comment:
I don't foresee the implications of multidomains keystone, but looks like that the recomended value for new installations  is indeed `false`.  If we do that, we need to take care that for an update (cloud 4 -> 5) we need to use the current value of `true`.